### ISSUE TITLE
Parse error when there're newlines in quotes

### ISSUE THREAD

**Issue by pythonissam** [bobjflong/yesod-csp#6]
Can we write like this:

```haskell
cspPolicy [csp|
  default-src 'self';
  script-src 'self' $nonce-rndVal https://cdnjs.cloudflare.com https://use.fontawesome.com https://code.jquery.com;
  style-src 'self' https://cdnjs.cloudflare.com https://cdn.jsdelivr.net https://code.jquery.com;
  img-src 'self' data:
|]
```

instead of:

```haskell
cspPolicy [csp|default-src 'self';
      script-src 'self' $nonce-rndVal https://cdnjs.cloudflare.com https://use.fontawesome.com https://code.jquery.com;
      style-src 'self' https://cdnjs.cloudflare.com https://cdn.jsdelivr.net https://code.jquery.com;
      img-src 'self' data:|]
```

**Comment by bobjflong** [comment]
Hey @pythonissam - I will take a look at this as soon as I can. Just to clarify - are you saying that the second example successfully compiles, but the first does not?

**Comment by pythonissam** [comment]
Yes, exactly.

**Comment by bobjflong** [comment]
Sorry for the extreme delay here. I believe this is fixed in 2.5.0 which I just pushed to hackage.
